tutorials/users/catalog/gui-search.md

```diff
@@ -1 +1,115 @@
-# gui-search
+# Catalog access via STAC Browser
+
+Each node in the Marble network include a GUI application called [STAC Browser](https://github.com/radiantearth/stac-browser) that allows you to 
+explore the STAC catalog of data that is available on that node. This application is available at `https://host-url/stac-browser`, where
+`host-url` is the URL of the Marble node. For example, for the Marble node 'Red Oak' available at 'https://daccs.cs.toronto.edu/',
+the STAC Browser can be accessed at <https://daccs.cs.toronto.edu/stac-browser>.
+
+In this tutorial, you will get an introduction to the STAC Browser, how to navigate within the Browser, how to perform search,
+and how to retrieve links that allow you to the data via one or more protocols. At the end of the section, we discuss some limitations with the GUI, 
+and programmatic way to interact with the STAC Catalog which is introduced in the [next section](pystac-client.ipynb).
+
+(gui_overview)=
+## GUI overview
+
+In this tutorial, we'll use Red Oak node as an example. The concepts introduced here apply to all STAC Browsers. 
+Start by navigating to the [STAC Browser on Red Oak](https://daccs.cs.toronto.edu/stac-browser) in a new tab; 
+you should see a screen similar to what is shown below:
+
+```{note}
+It is important to keep in mind that the data catalog on Marble is continuously evolving and so the catalogs and items that you see 
+may be different from that shown in this tutorial. However, the differences will only concern the 
+content that is displayed and not the STAC Browser itself. If any future changes to the interface or the functionality of the STAC Browser renders 
+aspects of this tutorial outdated, then the tutorial will be updated to reflect those changes.
+```
+
+![GUI Home Page](images/gui-home.png)
+
+1. The first thing you will notice is the bold text at the top left of the page. The STAC Browser uses this part of the page to display
+    the "title" of the page that is displayed. In this case, since you are at the root of the STAC Catalog, the appropriate title is the
+    _title_ field of the [root catalog](https://daccs.cs.toronto.edu/stac).
```

Comment:
Include in brackets the actual title to make it clear to the user (Data Analytics for Canadian Climate Services STAC API)